output/sqlite.go

```diff
@@ -0,0 +1,422 @@
+// +build cgo_sqlite
+
+package output
+
+import (
+	"bytes"
+	"database/sql"
+	"fmt"
+	"html/template"
+	"path/filepath"
+	"sort"
+	"strings"
+	"unicode"
+
+	_ "github.com/mattn/go-sqlite3"
+
+	"github.com/AdRoll/baker"
+)
+
+func init() {
+	All = append(All, []baker.OutputDesc{SQLiteDesc, SQLiteRawDesc}...)
+	sort.Slice(All, func(i, j int) bool { return All[i].Name < All[j].Name })
+}
```

Comment:
same as before, `init` is not needed.
But even if that was the case, I would let whoever needs `All` sorted the responsibility to sort it (in our case the help renderer)